src/api/useMovieGenresApi.js

```diff
@@ -0,0 +1,17 @@
+import axios from "../axios";
+import { MOVIE_GENRES } from "./endpoints.js";
+
+function useMovieGenresApi() {
+  async function retrieveMovieGenres() {
+    try {
+      const res = await axios.get(`/${MOVIE_GENRES}`);
+      return res.data;
+    } catch (error) {
+      console.log(error);
```

Comment:
If there is an error this function will return `undefined` but in MoviesPage I can see you expect an array.

I suggest adding explicit return type on the function and returning empty array if error occurs